Classes/Fusion/Eel/FlowQueryOperations/SortRecursiveByIndexOperation.php

```diff
@@ -29,14 +36,17 @@ class SortRecursiveByIndexOperation extends AbstractOperation
      */
     protected static $priority = 100;
 
+    #[\Neos\Flow\Annotations\Inject]
+    protected ContentRepositoryRegistry $contentRepositoryRegistry;
+
     /**
      * {@inheritdoc}
      *
      * We can only handle CR Nodes.
      */
     public function canEvaluate($context)
     {
-        return (isset($context[0]) && ($context[0] instanceof NodeInterface));
+        return (isset($context[0]) && ($context[0] instanceof Node)) || (is_array($context) && count($context) === 0);
```

Comment:
I fail to understand how the 2nd part of this change is related to Neos 9

Reply:
You are right, this is not related to Neos 9. Will revert this change.